Source/DafnyRuntime/DafnyRuntime.csproj

```diff
@@ -144,6 +144,9 @@
   <ItemGroup>
     <None Include="..\InterimKey.snk" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="DafnyRuntime.php" />
```

Comment:
Is this actually necessary? I'm guessing it is because it's not sitting in the Binaries directory as the JS and Go runtimes are?